includes/size/java.md

```diff
@@ -0,0 +1,2 @@
+!!!info "SDK bundle size"
+    You can get the SDK bundle size by checking [Maven repository](https://mvnrepository.com/artifact/com.amplitude). Select the version you are using and you can find size in the "Files" raw after 
```

Comment:
should this be https://mvnrepository.com/artifact/com.amplitude/java-sdk?
```suggestion
    You can get the SDK bundle size by checking [Maven repository](https://mvnrepository.com/artifact/com.amplitude/java-sdk). Select the version you are using and you can find the size in the "Files" row after.
```

Reply:
This file is shared by Java, Android and Kotlin. It looks like 
![image](https://github.com/Amplitude-Developer-Docs/amplitude-dev-center/assets/31029607/6bfad365-ea52-49da-853a-14a7f6d66616)

Reply:
should this be `row` instead of `raw`.

Reply:
updated